demo/pkg/subgraphs/test1/subgraph/schema.resolvers.go

```diff
@@ -247,12 +252,6 @@ func (r *Resolver) Subscription() generated.SubscriptionResolver { return &subsc
 type queryResolver struct{ *Resolver }
 type subscriptionResolver struct{ *Resolver }
 
-// !!! WARNING !!!
-// The code below was going to be deleted when updating resolvers. It has been copied here so you have
-// one last chance to move it out of harms way if you want. There are two reasons this happens:
-//   - When renaming or deleting a resolver the old code will be put in here. You can safely delete
-//     it when you're done.
-//   - You have helper methods in this file. Move them out to keep these resolver files clean.
 var (
 	aBigObject = &model.ABigObject{
 		AFieldOnABigObject: "a field on a big object - lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur excepteur sint occaecat cupidatat non proident sunt in culpa qui officia deserunt mollit anim id est laborum",
```

Comment:
This doesn't look right. It looks like someone committed resolver changes that are not in the schema. Does that var still serve a purpose?

Reply:
It's correct. Someone forgot to clean it up.